### DISCUSSION TITLE
Resize Ring entries without destroying and recreating the ring

### DISCUSSION THREAD

**Discussion by rch0wdhury** [axboe/liburing#845] [General]
Hi,

Is there any io_uring/liburing interface to Resize Ring entries without destroying and recreating the ring. For example, I start with io_uring_setup(20, &params) and then want to increase the SQ/CQ entries to 40.

Thanks,
Rajarshi.

**Comment by axboe** [comment]
No, it is currently not possible to resize the SQ or CQ ring once it has been setup.
